addons/survey/wizard/survey_invite_views.xml

```diff
@@ -23,15 +23,27 @@
                                  invisible="survey_access_mode != 'public'"/>
                             <field string="Send by Email" name="send_email" widget="boolean_toggle" options="{'autosave': False}"
                                    invisible="survey_access_mode != 'public'"/>
-                            <field name="partner_ids"
-                                widget="many2many_tags_email"
-                                placeholder="Add existing contacts..." options="{'no_quick_create': True}"
-                                context="{'show_email': True, 'form_view_ref': 'base.view_partner_simple_form'}"
-                                required="True"
-                                invisible="not send_email"/>
-                            <field name="emails"
-                                invisible="survey_users_login_required or not send_email"
-                                placeholder="e.g.  'Rick Sanchez' &lt;rick_sanchez@example.com&gt;, hictor_vugo@example.com"/>
+                        </group>
+                        <group invisible="not send_email">
+                            <group>
+                                <field name="partner_ids"
+                                    widget="many2many_tags_email"
+                                    placeholder="Add existing contacts..." options="{'no_quick_create': True}"
+                                    context="{'show_email': True, 'form_view_ref': 'base.view_partner_simple_form'}"
+                                    required="True"
+                                    invisible="not send_email"/>
+                            </group>
+                            <group>
+                                <field name="deadline" string="Block Participant after" placeholder="Deadline"/>
+                            </group>
+                        </group>
+                        <group name="emails" invisible="survey_users_login_required or not send_email">
+                            <label for="emails" string="Additional emails"/>
+                            <div name="custom_emails">
+                                <field name="emails"
+                                    placeholder="e.g.  'Rick Sanchez' &lt;rick_sanchez@example.com&gt;, hictor_vugo@example.com"/>
+                                <span invisible="not error" class="text-danger">Some emails you just entered are incorrect</span>
```

Comment:
Hey @mrsr-odoo,

Change the string as per the image mentioned in the log note
![Screenshot from 2024-05-28 18-32-44](https://github.com/odoo/odoo/assets/80447598/4386915e-8f5a-438d-92d8-353fdd838cd9)